src/open_inwoner/templates/cms/collaborate/active_plans_plugin.html

```diff
@@ -5,12 +5,13 @@ <h2 class="h2">
     {% button href="collaborate:plan_list" text=_("Naar samenwerken") icon="arrow_forward" icon_position="after" %}
 </h2>
 {% if plans %}
-<div class="plans-cards card-container card-container--columns-4">
+<div class="plans-cards card-container card-container--columns-4 card-container--user-home">
```

Comment:
Do we really need this new class here?I see it follows the same rules as the `card-container--columns-4`

Reply:
@jiromaykin to clean up and after that this PR can be merged

Reply:
@vaszig I will clean it up and request re-review.